docs: update the perception design document and the interface document (#499)
autowarefoundation/autoware-documentation: docs/design/autoware-interfaces/components/perception.md

### To Planning

Please refer to [the perception component design](../../autoware-architecture/perception/index.md#high-level-architecture) for detailed definitions of each output."

| Name               | Topic                                                   | Type                                                                                                                                                                     | Description                                                                                                                                                          |
| ------------------ | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Dynamic Objects    | `/perception/object_recognition/objects`                | [autoware_auto_perception_msgs/msg/PredictedObjects](https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/PredictedObjects.idl) | Set of dynamic objects with information such as a object class and a shape of the objects. Dynamic objects refer to those that were not present during map creation. |

Reviewer: ```suggestion
| Dynamic Objects    | `/perception/object_recognition/objects`                | [autoware_auto_perception_msgs/msg/PredictedObjects](https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/PredictedObjects.idl) | Set of dynamic objects with information such as a object class and a shape of the objects. These objects did not exist when the map was generated and are not contained within the map. |
```

Author: https://github.com/autowarefoundation/autoware-documentation/pull/499/commits/8454221efbf20ee523718608c51886e80beb7471
